app.js

```diff
@@ -168,7 +168,9 @@ app.start();
  * @param jvbJson
  */
 function getConferenceIds(jvbJson) {
-    return Object.keys(jvbJson.conferences);
+    // only report conferences that have rtcstats enabled.
+    return Object.keys(jvbJson.conferences)
+        .filter(confId => jvbJson.conferences[confId].enableRtcStats));
```

Comment:
Should we accept conferences that don't have the `enableRtcStats` key at all to maintain compatibility with older/current bridge versions?

Reply:
Right!